CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 1.27.0 - 5 October 2023
+### Added
+- Enhanced registries functionality: (https://github.com/microsoft/vscode-docker/issues/869)
+  - Allow third party providers to contribute to the registries view.
+  - Added GitHub Registries support.
+
+### Fixed
+- Resolved issues with opening .NET web apps in browsers built with .NET SDK containers. (https://github.com/microsoft/vscode-docker/issues/4006)
+- Updated Docker language server to address: 
```

Comment:
I'd make this a one-liner, i.e.

`- Fixed several Dockerfile language server issues. [#4056](...) ...`